src/igvfd/audit/measurement_set.py

```diff
@@ -335,3 +335,45 @@ def audit_missing_auxiliary_set_MPRA(value, system):
                 f'has no quantification DNA barcode sequencing `auxiliary_sets`.'
             )
             yield AuditFailure('missing auxiliary set', f'{detail} {description}', level='NOT_COMPLIANT')
+
+
+@audit_checker('MeasurementSet', frame='object')
+def audit_targeted_genes(value, system):
+    '''
+    [
+        {
+            "audit_description": "ChIP-seq and FISH-based assays are expected to specify targeted gene(s).",
+            "audit_category": "missing targeted genes",
+            "audit_level": "NOT_COMPLIANT"
+        },
+        {
+            "audit_description": "Only ChIP-seq and FISH-based assays are expected to specify targeted gene(s).",
+            "audit_category": "unexpected targeted genes",
+            "audit_level": "WARNING"
+        }
+    ]
+    '''
+    description_missing = get_audit_description(audit_targeted_genes, index=0)
+    description_unexpected = get_audit_description(audit_targeted_genes, index=1)
+    assay_term = value.get('assay_term')
+    assay_object = system.get('request').embed(assay_term, '@@object?skip_calculated=true')
+    assay_term_name = assay_object.get('term_name', '')
+    targeted_genes = value.get('targeted_genes', '')
+    preferred_assay_title = value.get('preferred_assay_title', '')
+    assays_expecting_targeted_genes = ['CRISPR perturbation screen followed by flow cytometry and FISH',
```

Comment:
This works fine, just thinking if it would make sense to split this list by `assay_term_name` & `preferred_assay_title` (seems a mixture here?), so it might be easier to maintain the list in future.